kiota_serialization_json/json_parse_node.py

```diff
@@ -203,7 +203,11 @@ def get_enum_value(self, enum_class: K) -> Optional[K]:
         raw_key = self.get_str_value()
         camel_case_key = None
         if raw_key:
-            camel_case_key = raw_key[0].upper() + raw_key[1:]
+            if raw_key == "none":
```

Comment:
is both None and none going to be problematic? in which case the comparison should be ignoring case.
Otherwise maybe update to None?

Reply:
Added logic to ignore case for "none" strings